star-sys/STAR/source/stitchAlignToTranscript.cpp

```diff
@@ -354,17 +377,13 @@ intScore stitchAlignToTranscript(uint rAend, uint gAend, uint rBstart, uint gBst
 
             for (uint ii=rBstart;ii<rBstart+L;ii++) Score+=scoreMatch; //add QS for mapped portions
 
-            Transcript trExtend;
```

Comment:
The constructor for `Transcript` seems to call `.reset()` so I'm unsure why there's a double call here.

Reply:
Most likey because the author didn't understand that that was what was happening.  Better question would be why the constructor wasn't just `Transcript() = default;` since the call to reset there doesn't actually do anything.